### PR TITLE
Add a liveness health probe in api server component 

### PR DESCRIPTION
**Pull request by odvarkadaniel**
Added /healtz endpoint and liveness probe to the deployment file.
cc @droslean

### CODE REVIEW

File: cluster/apiserver/apiserver.yaml [hunk 1]
```diff
@@ -72,3 +72,26 @@ items:
           image: ghcr.io/vega-project/vega-project/ccb-operator/apiserver:latest
           args:
           - --dry-run=false
+
+- apiVersion: v1
```
Comment (droslean) [inline cluster/apiserver/apiserver.yaml:76]: Please open a new PR for the liveness probe changes. Also, this is wrong. Here you create a pod that runs an image that we don't need. We already have the deployments [here](https://github.com/vega-project/ccb-operator/blob/master/cluster/apiserver/apiserver.yaml#L50-L74)

File: cmd/apiserver/main.go [hunk 2]
```diff
@@ -236,6 +236,10 @@ func main() {
 	router.HandleFunc("/calculation", s.getCalculation)
 	router.HandleFunc("/calculations/create", s.createCalculation)
 	router.HandleFunc("/calculations/delete/{id}", s.deleteCalculation)
+	router.HandleFunc("healthz", func(rw http.ResponseWriter, r *http.Request) {
```
Comment (droslean) [inline cmd/apiserver/main.go:239]: This is correct, but please open a separate PR.

File: cluster/apiserver/apiserver.yaml (hunk 1 above)
Comment (droslean) [reply cluster/apiserver/apiserver.yaml:76]: Oh, it is a separated PR, but you have the commits from the other PR as well.

Comment (odvarkadaniel) [reply cluster/apiserver/apiserver.yaml:76]: Yeah, I dont know why that happened...

**Comment by odvarkadaniel** [comment]
I am not really sure if It is possible to put it in like that. It shows 0 errors in the code.

File: cluster/apiserver/apiserver.yaml (hunk 1 above)
Comment (droslean) [reply cluster/apiserver/apiserver.yaml]: You did `git checkout -b liveness-prove` while you were in the `apiserver-tests` branch and you carried the commit from the `apiserver-tests` branch as well.

Comment (odvarkadaniel) [reply cluster/apiserver/apiserver.yaml]: Oh yes.. Totally forgot about that.

File: cluster/apiserver/apiserver.yaml [hunk 3]
```diff
@@ -72,3 +72,10 @@ items:
           image: ghcr.io/vega-project/vega-project/ccb-operator/apiserver:latest
           args:
           - --dry-run=false
+          metadata:
```
Comment (droslean) [inline cluster/apiserver/apiserver.yaml:75]: This is a leftover I assume?

Comment (odvarkadaniel) [reply cluster/apiserver/apiserver.yaml:75]: oh, I thought it was there already. Deleting it now.

**Pull request merged**